package.json

```diff
@@ -38,6 +38,7 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
+    "@ffmpeg-installer/ffmpeg": "^1.0.20",
```

Comment:
In the Docker image as well as in GitHub Actions we install ffmpeg by the operating system. If we add this NPM package it would result in two ffmpeg versions on the system. I personally would force the user to install ffmpeg on his system which is available in his PATH instead of using this package since it seems to be not very well maintained and "could" lead to unwanted behaviour.

Reply:
This is a good point. Long-term if we proceed with ffmpeg we'll likely get the binary from https://ffmpeg.org/download.html and  package it along with the chromium binaries. For linux we can also just use the system one and add it to the list of required deps to avoid duplication. Using @ffmpeg-installer/ffmpeg is the easiest way to validate the approach as it has binaries for all 3 platforms (2 yrs old but still working).

Reply:
This is a short-term solution for ffmpeg mainly to unblock Yury.
We'll bundle it with Chromium ASAP: https://github.com/microsoft/playwright/issues/3680

Reply:
fine for me then, sounds good to me 👍

Reply:
It works